Realm/RLMRealm.h

```diff
@@ -495,6 +496,22 @@ __deprecated_msg("Use `performMigrationForConfiguration:error:`") NS_REFINED_FOR
  */
 + (BOOL)performMigrationForConfiguration:(RLMRealmConfiguration *)configuration error:(NSError **)error;
 
+/**
+ Creates a `RLMThreadHandover` from the passed in objects that can be safely imported on a separate thread.
+
+ Used to pass objects between threads in a thread-safe manner.
+
+ @warning Note that the returned `ThreadHandover` must be imported at most once, and that the current version of the
+ Realm will remain pinned until this package is imported or deallocated.
+
+ @param objects The objects to pass between threads.
+ @return A package that can be imported on a separate thread.
+
+ @see RLMThreadHandover
+ */
+- (RLMThreadHandover *)exportThreadHandoverWithObjects:(NSArray<id<RLMThreadConfined>> *)objects
```

Comment:
This name doesn't fit the Cocoa naming scheme; that would be "threadHandoverWithObjects" or "exportThreadHandoverForObjects". I also think we should try to avoid using the word "handover" in the public API. While the concept of handing objects over to other threads is generally understood, the compound noun form of it is basically internal jargon, and perhaps more importantly what we offer *isn't* transferring an existing object to a different thread. Another issue with this name is that it suggests that it's more restricted than it actually is. The other places in our API where we refer to "Objects" we specifically mean RLMObject/RealmSwift.Objects (e.g. `addObjects:`).